### ISSUE TITLE
Min and max window size aren't provided to WM.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [mucommander/mucommander#957]
![image](https://github.com/mucommander/mucommander/assets/42837531/3d049e9e-0b4c-4814-986e-8636ec539bd9)

![image](https://github.com/mucommander/mucommander/assets/42837531/d5e570f3-4e80-4e26-a465-b162bbcb5763)

![image](https://github.com/mucommander/mucommander/assets/42837531/d6c199c7-45c8-4c02-9936-d53e96f99fac)

The problem is both vertical and horizontal, despite the screenshots mostly demonstrating one:

![image](https://github.com/mucommander/mucommander/assets/42837531/9d8a549a-7acc-4591-835d-703cc299ff30)

Providing a minimum size to the WM *is* possible - https://invent.kde.org/plasma/systemsettings does so: 

![image](https://github.com/mucommander/mucommander/assets/42837531/95edd403-b88d-4cc0-9e58-ef5925bcd5a8)

is its minimum size. See https://discuss.kde.org/t/1683?u=rokejulianlockhart for implementation information.

**Comment by ahadas** [comment]
@rokejulianlockhart you've mentioned that you use an rpm based operating system and it doesn't happen on my Fedora with GNOME so it's probably reproducible only with KDE - what operating system do you use?

**Comment by rokejulianlockhart** [comment]
@ahadas, I use https://en.opensuse.org/SDB:KDE_repositories#Adding_these_repos_to_an_existing_installation in [`cpe:/o:opensuse:tumbleweed:20230520`](cpe:/o:opensuse:tumbleweed:20230520). It's probably relevant to https://invent.kde.org/plasma/kwin, if this is not reproducible on https://forty.gnome.org/.

If you're *definitely* providing the minimum window size in a cross-DE manner, then I'll report this at bugs.kde.org on KWin (although I'd implore you to instead, since then you'll be the bug owner) but please first check some relevant reference code to verify that MuCommander does so already.

**Comment by rokejulianlockhart** [comment]
@ahadas, https://discuss.kde.org/t/where-in-plasma-systemsettings-does-it-define-its-minimum-window-size/1683/2?u=rokejulianlockhart

**Comment by ahadas** [comment]
@rokejulianlockhart the 'quit muCommadner' dialog also appears in #956, though the height there is too big, so I'm not sure it's the best test case but if it also looked as in the screenshot you've uploaded to this one, could you please check the [nightly build](https://github.com/mucommander/mucommander/releases/tag/nightly) to see if #959 improved it?

**Comment by rokejulianlockhart** [comment]
@ahadas, installing https://github.com/mucommander/mucommander/releases/download/nightly/mucommander-snapshot.x86_64.rpm has somehow deleted the `.desktop` file, but not created a new one.

What's the binary for https://github.com/mucommander/mucommander? I'll invoke it from $PATH.

**Comment by ahadas** [comment]
> @ahadas, [#959 (comment)](https://github.com/mucommander/mucommander/pull/959#issuecomment-1564695510)
> 
> > the 'quit muCommadner' dialog also appears in #956, though the height there is too big, so I'm not sure it's the best test case
> 
> _That_ would be most easily fixed with a maximum height size, too.

yeah, but if you keep seeing that the window is too small then it seems something is wrong with how swing is presented there. @rokejulianlockhart can you point to a Java application with UI is swing that works well there?
